.dockerignore

```diff
@@ -1,20 +0,0 @@
-*.css
```

Comment:
We are still building docker images from Amoc code, and we'll continue to do so. Why exactly did you remove the file?

Reply:
now it's `git clean -ffxd` executed before the build in the Dockerfile